### ISSUE TITLE
net461 folder not created

### ISSUE THREAD

**Issue by benpettman** [DataDog/dd-trace-dotnet#4540]
Have a .Net project running 4.6.1 used nuget to add the Datadog.Trace.Bundle to the solution.

Datadog folder is created, but build then proceeds to error with 

"error : Copying file datadog\net461\Datadog.Trace.AspNet.dll to obj\AWSDev\Package\PackageTmp\datadog\net461\Datadog.Trace.AspNet.dll failed. Could not find a part of the path 'datadog\net461\Datadog.Trace.AspNet.dll'"

Looking in the created datadog folder and there is no 'net461' folder created, so I am unsure how I am supposed to instrument the app?

**Comment by andrewlock** [comment]
Hi, I've just given this a test, and I can't seem to reproduce it 🤔 How are you doing the build and publish? What sort of application are you building? Which IDE are you using?

As a test, I created an ASP.NET .NET FX `net461` application in Visual Studio, added the `Datadog.Trace.Bundle` NuGet to the solution, and could build without any issues. I then did a publish, and the files were all copied to the correct place in the publish folder as expected.
